plugins/kotlin/idea/tests/test/org/jetbrains/kotlin/idea/inspections/AbstractLocalInspectionTest.kt

```diff
@@ -161,7 +163,8 @@ abstract class AbstractLocalInspectionTest : KotlinLightCodeInsightFixtureTestCa
         expectedProblemString: String?,
         expectedHighlightString: String?,
         localFixTextString: String?,
-        inspectionSettings: Element? = null
+        inspectionSettings: Element?,
+        noLocalFixTextStringList: List<String> = emptyList(),
```

Comment:
It's better not to add to variable names their type, for example, `List`. I renamed it to `noLocalFixTextStrings`.